src/cli/measure_manager.rb

```diff
@@ -468,24 +470,42 @@ def get_arguments_from_measure_info(measure_info)
       elsif type == "Double".to_OSArgumentType
         arg[:units] = argument.units.get if argument.units.is_initialized
         arg[:default_value] = argument.defaultValueAsDouble if argument.hasDefaultValue
-
+        if argument.hasDomain
+          min, max = argument.domainAsDouble
+          # I'm a bit wary of rounding issues... I think 1e308 instead of
+          # Float::MAX (1.7976931348623157e+308) is fine for our applications...
+          if min > -1e308 # -Float::MAX
+            arg[:min_value] = min
+          end
+          if max < 1e308 # Float::MAX
+            arg[:max_value] = max
+          end
+        end
       elsif type == "Quantity".to_OSArgumentType
         arg[:units] = argument.units.get if argument.units.is_initialized
         arg[:default_value] = argument.defaultValueAsQuantity.value if argument.hasDefaultValue
 
       elsif type == "Integer".to_OSArgumentType
         arg[:units] = argument.units.get if argument.units.is_initialized
         arg[:default_value] = argument.defaultValueAsInteger if argument.hasDefaultValue
-
+        if argument.hasDomain
+          min, max = argument.domainAsInteger
+          if min != -2147483648
```

Comment:
I'm confused what the purpose of this is? Same for above with 1e308.

Reply:
Same answer as above... When only mix or max is provided, one of the domain boundaries is going to be `std::numeric_limits<T>::lowest()` or `std::numeric_limits<T>::max()`.
In the case of an integer that's +- 2147483648, and for a double that's 1.7976931348623157e+308

Reply:
So if you did indeed set a min less than `lowest()` or a max greater than `max()`, wouldn't it not set/write those values?

Reply:
I don't see how you could do that though. It'd be an overflow error, but mostly you just can't do it,  bindings included.

cf:

```
$ irb
2.7.2 :002 > 1e308
 => 1.0e+308 
2.7.2 :003 > 1e309
 => Infinity 

```